provisioning/transport/amqp/src/ProvisioningTransportHandlerAmqp.cs

```diff
@@ -286,5 +302,15 @@ private void ValidateOutcome(Outcome outcome)
                 }
             }
         }
+
+        private static int jitterMax = 5;
+        private static int jitterMin = 0;
+        private static TimeSpan GenerateDelayWithJitterForRetry(TimeSpan defaultDelay)
```

Comment:
Please add a UT to test this.

Reply:
I can add a test to check that a single call for this random jitter falls within the expected bounds, but seeing as how the jitter relies on random number generation, I'm not sure how useful it will be.

Reply:
It's useful to check if it falls in a range and it's random :).